Aydin/backend/src/course/course_entity.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Entity,
+  Column,
+  PrimaryColumn,
+  ManyToOne,
+  ManyToMany,
+  JoinTable,
+  OneToMany,
+} from 'typeorm';
+import {Lecturer} from '../lecturer/lecturer_entity';
+import {Student} from '../student/student_entity';
+import {Lecture} from '../lecture/lecture_entity';
+
+@Entity()
+export class Course {
+  @PrimaryColumn()
+  CourseId: string;
```

Comment:
consistent spacing between fields please, i recommend putting two newlines (that is, one empty line) between each field